### PR TITLE
Project README.md spelling and phrasing adjustments

### PR DESCRIPTION
**Pull request by seanlowe**
Work for issue #89. Spent some time going over the README and adjusting phrasing so that it reads and flows better, as well as fixing typos that I came across.

First PR contribution for OSS, let me know how I did!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,145 +2,174 @@
...
+### Generate a Node.js app that is packed with best practices AND simplicity in mind. Based off our repo [Node.js best practices](https://github.com/goldbergyoni/nodebestpractices) (77,000 stars)
 
-❣️ **Alpha stage:** It's a very premature work-in-progress, we're only kicking-off now, please revisit by June 2022🗓
+<br />
 
-![Discord](/docs/images/discord-logo.png) [Discord discussions](https://discord.gg/9Nrarr7p) | ![Twitter](/docs/images/twitter-icon.png) [Twitter](https://twitter.com/nodepractices)
+❣️ **Alpha stage:** Practica.js is a work-in-progress. We've only recently kicked off. Please revisit by June 2022 🗓
 
-# What in here for you in one paragraph
+**Contact us:** ![Discord](/docs/images/discord-logo.png) [Chat with us on Discord](https://discord.gg/9Nrarr7p) | ![Twitter](/docs/images/twitter-icon.png) [Send us a Tweet](https://twitter.com/nodepractices)
```
Comment (goldbergyoni) [inline README.md:11]: The phrase "one paragraph" makes the point of how short it is, let's keep it?

File: README.md [hunk 2]
```diff
@@ -2,145 +2,174 @@
...
-  - More coming soon
-  </details>
-
-- [`Practices and features`](#practices-and-features)
-- [`Documentation site (coming soon)`](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-- [`YouTube channel (soon)`](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-- [`The people behind this project`](#the-people-behind-this-project)
+# `Table of Contents`
+
+- [`Installation and Usage`](#installation-and-usage)
```
Comment (goldbergyoni) [inline README.md:27]: Would prefer the "Very quick" phrasing (what we changed)

File: README.md [hunk 3]
```diff
@@ -2,145 +2,174 @@
...
+		
+		- Fastify, PostgreSQL
+		- Fastify, mongo-db
+		- Generate Your Own Interactively
+		- More coming soon
+		</details>
+
+<br />
+
+# Installation and Usage
```
Comment (goldbergyoni) [inline README.md:51]: Highly prefer punchy/bold language here, let's use "A very quick guide" or something that stands out. People tend to leave early unless they quickly find what they need

File: README.md [hunk 4]
```diff
@@ -2,145 +2,174 @@
...
+
+Keeping it simple, flat and based on native Node/JS capabilities is part of this project DNA. We believe that too many abstractions, high-complexity or fancy language features can quickly become a stumbling block for the team. 
 
-Keeping it simple, flat and based on native Node/JS capabilities is part of this project DNA. We believe that too many abstractions, high-complexity and fancy language features become soon a strategic slowdown for the team. To name a few example, our code flow is flat with almost no level of indirection, although using TypeScript - almost no features are being used besides types, for modularization we simply use... Node.js modules
+To name a few examples, our code flow is flat with almost no level of indirection, although using TypeScript - almost no features are being used besides types, for modularization we simply use Node.js modules
 
 ![Built on top of known frameworks](/docs/images/abstractions-vs-simplicity.png)
 
-### 3. Supports many technologies and frameworks
+### 3. Accessibility, supporting many technologies and frameworks
```
Comment (goldbergyoni) [inline README.md:118]: Prefer the origin text here

**Review by goldbergyoni — CHANGES_REQUESTED** [review]
Amazing contribution with *important* improvements! Many thanks 💚

There are fewer places where I prefer catching the visitor's attention with bold words over using 'correct' language, let's just revert those few examples and merge quickly?

File: README.md (hunk 1 above)
Comment (seanlowe) [reply README.md:11]: How does 'A One Paragraph Overview' sound?

File: README.md (hunk 2 above)
Comment (seanlowe) [reply README.md:27]: Gotcha. 

How attached to 'very quick' are we? It's difficult to make a title with the words 'very quick'. Not much works well with them unless it's in a full sentence. If we're wanting to emphasize the speed at which someone can get started, maybe we can do something with alliteration and call it a 'Speedy Setup' or, more generalized, something like 'Quick Start' / 'Rapid Start'. 

What do you think?

File: README.md (hunk 1 above)
Comment (goldbergyoni) [reply README.md:11]: Yes, love it

File: README.md (hunk 2 above)
Comment (goldbergyoni) [reply README.md:27]: My belief is that the visitor should read _unusual_ words. It should not sound too bad and lexically wrong, but we should find wording that is unusual and bold

I trust your judgment, put something, don't be afraid to deviate a bit from 'standard' English (e.g., Super-quick setup)

**Review by goldbergyoni — DISMISSED** [review]

**Review by goldbergyoni — APPROVED** [review]

**Comment by goldbergyoni** [comment]
@seanlowe This is an amaing PR. The wording is so much better and deliver our messages way better!

Once we have our t-shirts soon, I'll ping you to send one:)

Hope to see you here again, we're about to write a very thoughtful example app with tons of documentations as well

@all-contributors please add @seanlowe  for content

**Pull request merged**